packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.html

```diff
@@ -47,7 +47,7 @@
           result={result} content-url={contentURL}
           onquantic__loadingstatechange={handleLoadingStateChange}></c-quantic-quickview-content>
         <template if:false={isLoading}>
-          <footer onclick={stopPropagation} class="slds-modal__footer slds-p-top_none slds-p-bottom_none">
+          <footer onclick={stopPropagation} class="slds-modal__footer modal__footer slds-p-top_none slds-p-bottom_none">
```

Comment:
I would still respect the BEM notation for these classes: 
So keep the `quickview__` prefix on our classes.

```suggestion
          <footer onclick={stopPropagation} class="slds-modal__footer quickview__modal__footer slds-p-top_none slds-p-bottom_none">
```

Reply:
i can add quickview, but according to BEM you dont put 2 double underscore so it would be something like `quickview-modal__footer`

Reply:
Yea I was getting confused trying to give you the right value, I think we have two __ in many places even if that's not the standard lol.

But yes we need the prefix `quickview` at least